aea/crypto/cosmos.py

```diff
@@ -33,19 +33,20 @@
 
 import requests
 
-from aea.crypto.base import Crypto, LedgerApi
+from aea.crypto.base import Crypto, FaucetApi, LedgerApi
 from aea.mail.base import Address
 
 logger = logging.getLogger(__name__)
 
-COSMOS = "cosmos"
+_COSMOS = "cosmos"
```

Comment:
@marcofavorito I think the use of this constant was dangerous as it could lead to this being out of sync with what is set as `identifier` class var on `Crypto` (especially now with user defined types). I replaced it throughout.